Endzone.Umbraco.Extensions/PublishedContentExtensions/Collections.cs

```diff
@@ -89,5 +104,18 @@ public static IEnumerable<IEnumerable<T>> Split<T>(this IEnumerable<T> list, int
                          select part.AsEnumerable();
             return splits;
         }
+
+        /// <summary>
+        /// Returns the website settings node
+        /// </summary>
+        /// <param name="content"></param>
+        /// <returns></returns>
+        public static IPublishedContent GetWebsiteSettings(this IPublishedContent content)
```

Comment:
@barryokane @ondrejpialek  So we need to decide if we are going to use it on all websites or not. I have been adding it to all websites as I think it makes sense to keep general website information in one place and not clog up the Home page. 
I think it could go into a Settings or Configuration class?